### ISSUE TITLE
Ability to manually set a chat latency delay to sync with stream?

### ISSUE THREAD

**Issue by tlonelyg** [worldreboot/twitch-reloaded-roku#25]
Hopefully this will be seen:

It would be amazing if there was an option to simply delay chat by how long the video is delayed. It seems like the chat is shown live, but the video has a huge latency delay (which is fine). I think it's more important to focus on delaying the chat rather than reducing latency of the video. I just wish there was a way to manually delay the chat to sync with the video, maybe a spot in the settings to manually set it to be delayed by seconds or milliseconds.

Twoku is honestly an amazing app and this is the only problem I find myself running into. It's the only thing making me want to buy an amazon fire stick or something like that with integrated twitch. But yeah, great work! Would love to see this being added in the future.

**Comment by worldreboot** [comment]
Yeah, this feature has been highly requested since chat was added to the app. Someone was actually working on it, but I forget the reason why they stopped. I (someone?) might continue from where they left off soon. Thanks

**Comment by tlonelyg** [comment]
Glad it is being looked into, thanks!
